templates/taxi/car_detail.html

```diff
@@ -18,5 +18,19 @@ <h1>Drivers</h1>
     {% for driver in car.drivers.all %}
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
-  </ul>
+  </ul> 
+  
+  <form action="" method="post" novalidate>    
+    {% csrf_token %}
+
+    {% if user not in car.drivers.all %}
```

Comment:
Ensure that the logic `{% if user not in car.drivers.all %}` correctly determines if the current user is associated with the car. This logic is crucial for displaying the correct form fields.